src/session.c

```diff
@@ -828,12 +835,16 @@ static CK_RV token_login(P11PROV_SESSION *session, P11PROV_URI *uri,
 
 done:
     OPENSSL_cleanse(cb_pin, cb_pin_len);
+    P11PROV_debug("returning 0x%x", ret);
```

Comment:
don't think we need this, we should have debug in the caller if ret is an error.